src/stores/Frame/DistanceMeasuringStore.ts

```diff
@@ -6,36 +6,43 @@ import {getColorForTheme} from "utilities";
 import {Point2D, Transform2D} from "models";
 
 export class DistanceMeasuringStore {
-    start: Point2D;
-    finish: Point2D;
+    @observable start: Point2D;
+    @observable finish: Point2D;
 
     @observable transformedStart: Point2D;
     @observable transformedFinish: Point2D;
     @observable isCreating: boolean;
     @observable color: string;
+    @observable lineWidth: number;
+    @observable fontSize: number;
 
     static readonly DEFAULT_WIDTH = 1.5;
     static readonly DEFAULT_FONTSIZE = 14;
     static readonly DEFAULT_COLOR = "#62D96B";
 
     constructor() {
         makeObservable(this);
-        this.transformedStart = {x: null, y: null};
-        this.transformedFinish = {x: null, y: null};
+        this.start = {x: 0, y: 0};
```

Comment:
shall we also initialize start/finish to NaNs?

Reply:
That way, the values shown on the input section will be NaN when users first open the dialog, rather than 0.

Reply:
I see... maybe both is ok

Reply:
Do you mean keep the initialization value to 0?

Reply:
Yes we can keep it